code/game/machinery/telecomms/telecomunications.dm

```diff
@@ -170,6 +171,11 @@ var/global/list/obj/machinery/telecomms/telecomms_list = list()
 	telecomms_list -= src
 	..()
 
+/obj/machinery/telecomms/unlinkFrom(var/mob/user, var/obj/buffer)
+	..()
+	for(var/obj/machinery/computer/telecomms/monitor/M in range(25,src))
```

Comment:
Is 25 a magical number or is there a reason for this specific range?

Reply:
25 tiles is the distance in which the network monitor will be able to pick up nearby machines. Contrary to what you might expect it does not simply add the machines that are on its network (as though it were copying over a list). Instead, it searches for all telecoms equipment inside 25 then checks to see if its network var matches.

So for example the network monitor won't display the mining relay even though it's on the tcommsat network

Reply:
Understandable